include/nanobind/eigen/sparse.h

```diff
@@ -131,7 +131,7 @@ template <typename T> struct type_caster<T, enable_if_t<is_eigen_sparse_matrix_v
         StorageIndexNDArray inner_indices(src->innerIndexPtr(), 1, data_shape, owner);
 
         try {
-            return matrix_type(make_tuple(
+            return matrix_type(nanobind::make_tuple( // FIXME: better way to suppress ADL
```

Comment:
I'm not sure whether this is acceptable. Maybe there is a nicer way to resolve the ambiguous call?